### DISCUSSION TITLE
Better communication from the TSC to the community

### DISCUSSION THREAD

**Discussion by modir** [laminas/technical-steering-committee#170] [Ideas]
Just by accident I found out now what this committee is doing and I think many others feel the same. It would be great if the community is informed better about what is going on in the meetings. Hence I propose what I already see happening in several other open source projects:

Please put the meeting notes into the community forums on [discourse](https://discourse.laminas.dev/). You can then inform all registered users about the new meeting notes. This way we as "normal" developers see what is going on at the TSC plus we can give feedback. This way you can then motivate more people to become active within the project as well.

**Comment by froschdesign** [comment]
@modir 
> Please put the meeting notes into the community forums on [discourse](https://discourse.laminas.dev/).

Discourse is not the central platform of this project and in my opinion, only the results of the meetings should receive further publication, but not the entire conversation.

**Comment by modir** [comment]
> Discourse is not the central platform of this project and in my opinion, only the results of the meetings should receive further publication, but not the entire conversation.

What is then the central platform of this project at the moment? I think this is the main problem. There is non. Hence it is not clear where to find what information.

**Comment by froschdesign** [comment]
At the moment it is GitHub.
But for the results of the meetings it should/can be the blog.
